### ISSUE TITLE
The jypyter environment should start in humlabspeech/Applications

### ISSUE THREAD

**Issue by FredrikKarlssonSpeech** [humlab-speech/visible-speech-deployment#53]

**Comment by johanvonboer** [comment]
@FredrikKarlssonSpeech Does this mean that the humlabspeech_emuDB directory should also be in Applications? Or should that stay in Data?

**Comment by FredrikKarlssonSpeech** [comment]
No. We just need the jypyter startup directory to be in Applications.

Data <- here we store the data
Applications <- Here we store analysis code (including notebooks, R scripts and such things)
Results <- Here we store results of code (Figures, Excel files and so on).

I am assuming now that  the Data directory still is accessible as ../Data/ if jypyter starts in Applications? 
Otherwise, we need to think about this more..

**Comment by johanvonboer** [comment]
Makes sense!

Yes I agree the user should of course have access to their entire project in Jupyter, not just Applications. Will look into setting this up.

**Comment by TomasSkotare** [comment]
Does this work as expected now?

**Comment by johanvonboer** [comment]
Should be fixable simply by setting:

`c.NotebookApp.notebook_dir = '/home/jovyan/project/Applications'`

In the jupyter_notebook_config.py file, but in order to do that I need to rebuild the jupyter image and that's currently not possbile due some sort of build error that has suddenly popped up. Tomas is working on it.

**Comment by johanvonboer** [comment]
Previous Johan from 2021 was right, setting `c.NotebookApp.notebook_dir = '/home/jovyan/project/Applications'`will lock the user to the directory...
